### PR TITLE
Add job start/end time to job status

### PR DESCRIPTION
**Pull request by HecarimV**
ref:
volcano-sh/volcano#1750

### CODE REVIEW

File: pkg/apis/batch/v1alpha1/job.go [hunk 1]
```diff
@@ -317,6 +317,14 @@ type JobStatus struct {
 	// The resources that controlled by this job, e.g. Service, ConfigMap
 	// +optional
 	ControlledResources map[string]string `json:"controlledResources,omitempty" protobuf:"bytes,11,opt,name=controlledResources"`
+
+	// The time when the job starts running
+	// +optional
+	JobStartTime metav1.Time `json:"jobStartTime,omitempty" protobuf:"bytes,12,opt,name=jobStartTime"`
```
Comment (shinytang6) [inline pkg/apis/batch/v1alpha1/job.go:323]: l think `StartTime` is better, and `metav1.Time` should be pointer.

File: pkg/apis/batch/v1alpha1/job.go [hunk 2]
```diff
@@ -317,6 +317,14 @@ type JobStatus struct {
...
+
+	// The time when the job phase turn to completed or failed
+	// +optional
+	JobEndTime metav1.Time `json:"jobEndTime,omitempty" protobuf:"bytes,13,opt,name=jobEndTime"`
```
Comment (shinytang6) [inline pkg/apis/batch/v1alpha1/job.go:327]: ditto

**Review by shinytang6 — CHANGES_REQUESTED** [review]

**Comment by shinytang6** [comment]
And btw please fix the DCO and maybe you should run `./hack/update-codegen.sh` to update the deepcopy file )

**Comment by k82cn** [comment]
Let's add a `Conditions[]` to `JobStatus`, similar to `Conditions[]` of `PodStatus`

**Comment by HecarimV** [comment]
> And btw please fix the DCO and maybe you should run `./hack/update-codegen.sh` to update the deepcopy file )

Thanks, I'll check above.

**Comment by HecarimV** [comment]
> Let's add a `Conditions[]` to `JobStatus`, similar to `Conditions[]` of `PodStatus`

Okay!  I'll reconsider the struct, thanks.

**Comment by Thor-wl** [comment]
BTW,please fix the DCO

File: pkg/apis/batch/v1alpha1/job.go (hunk 1 above)
Comment (Thor-wl) [reply pkg/apis/batch/v1alpha1/job.go]: Agree with @shinytang6

**Pull request closed without merging**